web/src/hooks/arrows.ts

```diff
@@ -11,6 +11,19 @@ import exchangesToExclude from '../../config/excluded_aggregated_exchanges.json'
 const exchangesConfig: Record<string, any> = exchangesConfigJSON;
 const { exchangesToExcludeZoneView, exchangesToExcludeCountryView } = exchangesToExclude;
 
+function shouldHideExchangeIntensity(
+  exchange: string,
+  zonesWithOutages: string[],
+  value: number
+) {
+  const [zone1, zone2] = exchange.split('->');
+
+  return (
+    (zonesWithOutages.includes(zone1) && value > 0) ||
+    (zonesWithOutages.includes(zone2) && value < 0)
+  );
+}
+
```

Comment:
I had a little hard time understanding what this was supposed to do at first glance. Could you maybe add a comment and or a test for our future selfs?

Reply:
Thanks for the feedback, added some comments and tests now :)